File: Python/Gpu.py
from Komponen import Komponen

class Gpu(Komponen):
    def __init__(self, merk, nama, memori_gb):
        super().__init__(merk, nama)
        self.__memori_gb = memori_gb

    # Getter
    def get_memori_gb(self):
        return self.__memori_gb

    # Setter
    def set_memori_gb(self, memori_gb):
        self.__memori_gb = memori_gb

File: Cpp/Gpu.cpp
#pragma once
#include <iostream>
#include <string>
#include "Komponen.cpp"

using namespace std;

class Gpu : public Komponen {
private:
    int vramGB;
public:
    Gpu() {}

    Gpu(int vramGB, string merk, string nama) 
        : Komponen(merk, nama) {
        this->vramGB = vramGB;
    }

    void setVramGB(int vramGB) {
        this->vramGB = vramGB;
    }

    int getVramGB() const { 
        return this->vramGB;
    }
    

    ~Gpu() {}
};
